### PR TITLE
[ADD] payment_acquirers/adyen: add doc configuration of Adyen

### PR DESCRIPTION
**Pull request by chevalierv**
Following a technical task regarding the migration of the Adyen hosted
payment page integration to Adyen PayByLink, some new parameters have
been introduced and some have been removed.

This new addition to the documentation will help users configure their
acquirer.

See also:
- https://github.com/odoo/odoo/pull/87041

task-2802921

### CODE REVIEW

**Review by LoredanaLrpz — CHANGES_REQUESTED** [review]
Hey @chevalierv, 
Could you please place your images in the correct folder and name them according to the doc [https://www.odoo.com/web#id=2823414&cids=1&menu_id=4720&action=333&active_id=1623&model=project.task&view_type=form](url) (separated with hyphens (-) and in a folder named after the doc followed by _) (adyen_add-webhook ; adyen_api-urls ; ...) 

@jcs-odoo, can you please confirm the names proposed are correct? 
Thanks! :)

**Review by LoredanaLrpz — APPROVED** [review]

**Comment by jcs-odoo** [comment]
> Hey @chevalierv, Could you please place your images in the correct folder and name them according to the doc [https://www.odoo.com/web#id=2823414&cids=1&menu_id=4720&action=333&active_id=1623&model=project.task&view_type=form](url) (separated with hyphens (-) and in a folder named after the doc followed by _) (adyen_add-webhook ; adyen_api-urls ; ...)
> 
> @jcs-odoo, can you please confirm the names proposed are correct? Thanks! :)

Sorry, I didn't reply. The media filenames seem fine indeed :)

File: content/applications/finance/payment_acquirers.rst [hunk 1]
```diff
@@ -81,7 +83,8 @@ Online Payment Providers
 +-------------------------------------+----------------------+-----------+------------+-----------------+
 | SIPS                                | ✔                    |           |            |                 |
 +-------------------------------------+----------------------+-----------+------------+-----------------+
-| Stripe                              | ✔                    | ✔         | ✔          |                 |
+| :doc:`Stripe                        | ✔                    | ✔         | ✔          |                 |
+| <payment_acquirers/stripe>`         |                      |           |            |                 |
```
Comment (Demesmaeker) [inline content/applications/finance/payment_acquirers.rst:87]: Why is this change needed for the Adyen migration to HPP?

Comment (chevalierv) [reply content/applications/finance/payment_acquirers.rst:87]: My intention was to add a link to the doc of the acquirer (if it exists in this version). But I see that I spoil a bit of master here :3

Comment (Demesmaeker) [reply content/applications/finance/payment_acquirers.rst]: If you want to revamp the doc en V13.0, please do so!!! =D

... But maybe not with a task called "Migrate Adyen to HPP" :smile:

**Comment by jcs-odoo** [comment]
@chevalierv  I see here another PR about Adyen by @toaa-odoo in 15.0 #2241 
Could you both check what needs to be done where in which version?
Thanks

**Comment by toaa-odoo** [comment]
I just compared both versions, and IMO both are missing information. #2241 

@chevalierv The backend of Adyen changed a little, and the path **Additional** Settings ‣ HMAC Key ‣ Generate new HMAC **key** (for example) is no longer the correct one, the category names are now **Security** ‣ HMAC Key ‣ **Generate** (see my PR). The URL name also changed from **Recurring API** to **Classic Recurring API**.

On the other hand, my PR does not feature the **Server Configuration ‣ Method, select HTTP Post** step and could certainly feature it. Also, my PR mentions the need to pick a recurring URL, but yours does not, so I'm not sure if it's actually required.

Let's discuss this on Discord, I can add your info to my PR and then push it! :) @toaa-odoo

File: content/applications/finance/payment_acquirers/adyen/update.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
+=================================
+Install the Adyen PayByLink Patch
+=================================
+
+Starting October 2022, due to a change in their API, :doc:`Adyen <../adyen>` needs to be updated to
+continue to accept payment.
```
Comment (AntoineVDV) [inline content/applications/finance/payment_acquirers/adyen/update.rst:6]:
```suggestion
continue accepting payments.
```

File: content/applications/finance/payment_acquirers/adyen/update.rst [hunk 3]
```diff
@@ -0,0 +1,54 @@
+=================================
+Install the Adyen PayByLink Patch
+=================================
+
+Starting October 2022, due to a change in their API, :doc:`Adyen <../adyen>` needs to be updated to
```
Comment (AntoineVDV) [inline content/applications/finance/payment_acquirers/adyen/update.rst:5]: Rather than evoking "a change in their API" (not very clear), let's directly link readers to the official announcement: https://docs.adyen.com/online-payments/classic-integrations/hosted-payment-pages/deprecation

**Review by AntoineVDV — CHANGES_REQUESTED** [review]

**Review by AntoineVDV — APPROVED** [review]

**Comment by AntoineVDV** [comment]
@robodoo r+
